core/test/agents/llm_agent_test.ts

```diff
@@ -129,9 +129,9 @@ describe('LlmAgent.callLlm', () => {
     const responses: LlmResponse[] = [];
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     for await (
-        const response of (agent as any)
```

Comment:
any reason to change this, if no please revert to reduce number of changed lines for the review.